artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
```

Comment:
Having a batch mode that gives you _all_ APIs has a different value because it opts in new APIs by default.